packages/calcite-components/src/components/tile/tile.scss

```diff
@@ -94,25 +117,31 @@
 }
 
 .description {
-  color: var(--calcite-tile-description-text-color);
+  color: var(--calcite-tile-description-text-color, var(--calcite-internal-tile-description-text-color));
```

Comment:
this should reference `--calcite-tile-text-color`

Reply:
The UI Kit shows different colors for heading and description:
<img width="650" alt="image" src="https://github.com/user-attachments/assets/fc08b2ce-3515-44c9-953d-b39c24836c40">
<img width="644" alt="image" src="https://github.com/user-attachments/assets/518adc71-bf44-4320-a20a-b82da73f9dcb">